src/devices/BaseDevice.tsx

```diff
@@ -123,6 +123,10 @@ export default class BaseDevice {
     this._notifyLogListeners(entry);
   }
 
+  getProcessName(pid: number): Promise<string> {
+    return Promise.reject(() => new Error('Not supported'));
+  }
+
```

Comment:
If I understand correctly, this will break the logs plugin for all device types other than android, e.g. iOS.

You could fix it by returning 'unknown-process' or similar.

Reply:
Got it. Will change it.